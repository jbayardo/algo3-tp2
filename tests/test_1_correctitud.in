11
0 10; 0 1; 1 2; 2 3; 3 4; 4 5; 5 6; 6 7; 7 8; 8 9; 9 10
11
0 1; 1 2; 2 3; 3 4; 4 5; 5 6; 6 7; 7 8; 8 9; 9 10
11
0 1; 0 5; 1 2; 2 3; 3 4; 4 6; 5 7; 6 8; 7 10; 8 9
11
0 1; 0 5; 1 2; 2 3; 3 4; 4 6; 5 7; 6 7; 6 8; 7 10; 8 9
8
0 3; 1 2; 3 4; 3 5; 5 6; 5 7
7
0 1; 0 2; 0 3; 1 2; 2 3; 2 4; 2 5; 2 6; 4 5; 5 6
13
0 1; 0 2; 0 3; 0 7; 1 2; 2 3; 2 4; 2 5; 2 6; 4 5; 5 6; 6 7; 7 8; 7 12; 8 9; 9 10; 10 11