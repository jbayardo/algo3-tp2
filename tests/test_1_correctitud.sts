Excercise 1 Timer			5,4,4,3,2,2,5
