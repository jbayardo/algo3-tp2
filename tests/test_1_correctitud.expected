10
10
3
7
3
5
7